src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -2912,9 +3053,9 @@ def test_resource_policyexemption_subscription(self, resource_group):
         if not self.in_recording:
             with mock.patch('azure.cli.command_modules.resource.custom._get_subscription_id_from_subscription',
                             return_value=MOCKED_SUBSCRIPTION_ID):
-                self.resource_policyexemption_operations(resource_group, None, '0b1f6471-1bf0-4dda-aec3-cb9272f09590')
+                self.resource_policyexemption_operations(resource_group, None, 'f67cc918-f64f-4c3f-aa24-a855465f9d41')
         else:
-            self.resource_policyexemption_operations(resource_group, None, '0b1f6471-1bf0-4dda-aec3-cb9272f09590')
+            self.resource_policyexemption_operations(resource_group, None, 'f67cc918-f64f-4c3f-aa24-a855465f9d41')
```

Comment:
Please do not modify the subscription ID we use, because it will cause our live test to fail because our team don't have permission